instances.go

```diff
@@ -171,9 +175,10 @@ func (i *Instance) UnmarshalJSON(b []byte) error {
 
 // GetUpdateOptions converts an Instance to InstanceUpdateOptions for use in UpdateInstance
 func (i *Instance) GetUpdateOptions() InstanceUpdateOptions {
+	group := i.Group
 	return InstanceUpdateOptions{
 		Label:           i.Label,
-		Group:           i.Group,
+		Group:           &group,
```

Comment:
I thought this would affect the value of the `group` string in the original instance, but now I realize a `string` in Go is immutable. Thanks for the suggestion!

Reply:
GitHub doesn't allow me to commit this suggestion, I will push one from my local branch.
<img width="606" alt="image" src="https://github.com/linode/linodego/assets/121905282/ba350b16-3561-4919-9dfe-8092ed29572e">